python/PyQt/PyQt/QtWidgets.py.in

```diff
@@ -21,12 +21,49 @@ __author__ = 'Matthias Kuhn'
 __date__ = 'November 2015'
 __copyright__ = '(C) 2015, Matthias Kuhn'
 
-from PyQt@QT_VERSION_MAJOR@.QtCore import QT_VERSION
 from PyQt@QT_VERSION_MAJOR@.QtWidgets import *
 
 QLayout.setMargin = lambda self, m: self.setContentsMargins(m, m, m, m)
 
-if (QT_VERSION >= 0x060000):
+if @QT_VERSION_MAJOR@ == 6:
   from PyQt6.QtGui import QAction as _QAction, QShortcut as _QShortcut
   QAction = _QAction
   QShortcut = _QShortcut
+
+  # patch back in Qt flags removed in PyQt
+  QWidget.RenderFlags = lambda flags=0: QWidget.RenderFlag(flags)
+  QGraphicsItem.GraphicsItemFlags = lambda flags=0: QGraphicsItem.GraphicsItemFlag(flags)
+  QAbstractItemView.EditTriggers = lambda flags=0: QAbstractItemView.EditTrigger(flags)
+  QAbstractSpinBox.StepEnabled = lambda flags=0: QAbstractSpinBox.StepEnabledFlag(flags)
+  QColorDialog.ColorDialogOptions = lambda flags=0: QColorDialog.ColorDialogOption(flags)
+  QStyle.State = lambda flags=0: QStyle.StateFlag(flags)
+  QStyle.SubControls = lambda flags=0: QStyle.SubControl(flags)
+  QDateTimeEdit.Sections = lambda flags=0: QDateTimeEdit.Section(flags)
+  QDialogButtonBox.StandardButtons = lambda flags=0: QDialogButtonBox.StandardButton(flags)
+  QDockWidget.DockWidgetFeatures = lambda flags=0: QDockWidget.DockWidgetFeature(flags)
+  QFileDialog.Options = lambda flags=0: QFileDialog.Option(flags)
+  QFontComboBox.FontFilters = lambda flags=0: QFontComboBox.FontFilter(flags)
+  QFontDialog.FontDialogOptions = lambda flags=0: QFontDialog.FontDialogOption(flags)
+  QGestureRecognizer.ResultFlag = lambda flags=0: QGestureRecognizer.ResultFlag(flags)
```

Comment:
that one looks suspicious given https://doc.qt.io/qt-6/qgesturerecognizer.html#ResultFlag-enum
Shouldn't that be:
```suggestion
  QGestureRecognizer.Result = lambda flags=0: QGestureRecognizer.ResultFlag(flags)
```

Reply:
ah yes, good catch